docs/sections/calculations/workflows.md

```diff
@@ -131,5 +119,32 @@ PwBandsWorkChain<113> Finished [0] [7:results]
 The bracket `[7:result]` indicates the current step in the outline of the `PwBandsWorkChain` (step 7, with name `result`).
 The `process status` is particularly useful for debugging complex work chains, since it helps pinpoint where a problem occurred.
 
-Congratulations on finishing the first part of the tutorial!
-In the next section, we'll look at how to organize and query your data.
+
+## Displaying the results
+
+Once the work chain has finished running, use `verdi process show <PK>` to inspect the `PwBandsWorkChain` and find the PK of its `band_structure` output.
+We can also plot the band structure using the `verdi shell`:
+
+```{code-block} console
+
+$ verdi data bands export --format mpl_pdf --output band_structure.pdf <PK>
+
+```
+
+Use the `evince` command or the JupyterHub file manager to open the `band_structure.pdf` file.
+It should look similar to the one shown here:
+
+:::{figure} include/images/si_bands.png
+:width: 100%
+
+Band structure computed by the `PwBandsWorkChain`.
+
+:::
+
+:::{important} **What we learnt**
```

Comment:
Or maybe something less demeaning? Like "What this tutorial teaches/has teached you" or "Summary of key points in this section/tutorial"?
```suggestion
:::{important} **What you learnt**
```

Reply:
Haha demeaning? I agree one needs to be careful with the general tone but I don't think people will get offended by a title like that.

Anyways, what do you think of

```suggestion
:::{important} **Key Takeaways**
```

Reply:
I like it 👍 But maybe lose the capitalization of `Takeaways`?